File: min-max sum/code.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'miniMaxSum' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#
def miniMaxSum(arr):
        minn = arr[0]
        maxx = arr[0]
        summ = 0
        for i in range(0,5):
            if arr[i] > maxx:
             maxx = arr[i]

            if arr[i] < minn:
             minn = arr[i]
            summ=summ+arr[i]
        print(summ-maxx,summ-minn)

if __name__ == '__main__':

    arr = list(map(int, input().rstrip().split()))

    miniMaxSum(arr)

File: min-max sum/code.rb
#!/bin/ruby

require 'json'
require 'stringio'

#
# Complete the 'miniMaxSum' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#

# def miniMaxSum(arr)
#     min = arr.sum - arr.max
#     max = arr.sum - arr.min

#     puts "#{min} #{max}"

# end
def miniMaxSum(arr)
  total = arr.reduce(&:+)
  min = max = nil
  # min = arr.min
  # max = arr.max
  arr.each do |i|
    if min == nil || i < min
      min = i
    end
    if max == nil || i > max
      max = i
    end
  end
  puts "#{total - max} #{total - min}"
end
arr = gets.rstrip.split.map(&:to_i)

miniMaxSum arr
